wallet/src/de/schildbach/wallet/ui/CurrencyCalculatorLink.java

```diff
@@ -1,212 +0,0 @@
-/*
- * Copyright 2013-2015 the original author or authors.
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-package de.schildbach.wallet.ui;
-
-import javax.annotation.Nullable;
-
-import org.bitcoinj.core.Coin;
-import org.bitcoinj.utils.ExchangeRate;
-import org.bitcoinj.utils.Fiat;
-
-import de.schildbach.wallet.Constants;
-
-import org.dash.wallet.common.ui.CurrencyAmountView;
-import org.dash.wallet.common.ui.CurrencyAmountView.Listener;
-
-import android.view.View;
-
-/**
- * @author Andreas Schildbach
- */
-public final class CurrencyCalculatorLink {
```

Comment:
Not used

Reply:
Right, this was used in v6 and prior.  there were two places to enter an amount.  One was in DASH, the other in the local currency and this linking class could change the other based on the exchange rate.